smac/runhistory/runhistory2epm.py

```diff
@@ -129,16 +131,16 @@ def __init__(
                              type(self.imputor))
 
         # Learned statistics
-        self.min_y = None
-        self.max_y = None
-        self.perc = None
+        self.min_y = np.NaN
```

Comment:
Doesn't this impact line 206 in this file?

Reply:
Good catch!